packages/css-data/src/shorthands.test.ts

```diff
@@ -1448,3 +1448,7 @@ test("expand position-try", () => {
     ["position-try-options", "--dashed-ident"],
   ]);
 });
+
+test("Does not fail on empty transition", () => {
+  expect(expandShorthands([["transition", ""]])).toEqual([]);
```

Comment:
weird one

Reply:
I think any value can be empty, we need to test it with empty string for all properties